src/actions/navigation.js

```diff
@@ -10,14 +10,21 @@ const makeDataPathFromParsedParams = (parsedParams) => {
   return tmp_levels.map((d) => d[1]).join("_");
 };
 
+export const makeDataPathFromPathname = (pathname) => {
+  return pathname
+    .replace(/^\/+/, '')    // strip leading
+    .replace(/\/+$/, '')    //   and trailing slashes
+    .replace(/\/+/g, '_');  // replacing all internal ones with underscores
+};
+
 /* match URL pathname to datasets (from manifest) */
 export const getDatapath = (pathname, availableDatasets) => {
   if (!availableDatasets) {return undefined;}
   const parsedParams = parseParams(pathname, availableDatasets);
   if (parsedParams.valid) {
     return makeDataPathFromParsedParams(parsedParams);
   }
-  return pathname.replace(/^\//, '').replace(/\/$/, '').replace(/\//g, '_');
+  return makeDataPathFromUrl(pathname);
```

Comment:
This is undefined - do you mean `makeDataPathFromPathname`?

Reply:
Yes, I did!  I thought I had tested this, but I now realize I was actually testing the `return` two lines up on line 25. My test cases were:

1. `/zika` and `/flu/seasonal/h3n2/ha/3y` (and variants) *without* a manifest present
2. `/zika` and `/flu/seasonal/h3n2/ha/3y` (and variants) *with* a manifest present

but it should have also included:

3. Some dataset _not_ in the manifest but loaded *with* a manifest present

I'll fix this up.

Reply:
(The reason for the name difference is that I had originally chosen `makeDataPathFromUrl`, but then adjusted it to parallel `makeDataPathFromParsedParams`.)